requirements/base.txt

```diff
@@ -2,7 +2,7 @@ chevron~=0.12
 click~=7.1
 Flask~=1.1.2
 #Need to add Schemas latest SDK.
-boto3~=1.14
+boto3~=1.18
```

Comment:
Haven't experienced any conflicts

Reply:
Why do we want to upgrade boto3?

Reply:
`DisableRollback` parameter hadn't been added to `execute_change_set` in version 1.14:
https://boto3.amazonaws.com/v1/documentation/api/1.14.0/reference/services/cloudformation.html?highlight=execute_change_set#CloudFormation.Client.execute_change_set

Reply:
Can we confirm what the past issues were that we had with bumping the boto version and make sure we aren't seeing those?

Reply:
I'm not sure what issues we were experiencing before, but I'll ask Mehment. There's a comment about a bug in  `dev.txt`, but I think that has been resolved.